qiskit/circuit/timestep.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
I thought we agreed to make this a separate PR? @itoko could you remove this?

Reply:
Done at e2a7c02